FreeRTOS/Test/CMock/tasks/tasks_utest_1.c

```diff
@@ -0,0 +1,5117 @@
+/*
```

Comment:
This is a lot of test cases in a single file.

Can you add some "@coverage" tags with the functions you are calling so that the lcov and lcovhtml targets work?

Reply:
i am calling all the functions in this file
basically the difference between files is the configuration options not different functions

Reply:
@alfred2g Then please put a single comment at the top of the file so that filtercov.py won't filter out all of your coverage data.

The point of coverage filtering is to make sure you're not taking credit for incidental coverage. It's not effective when every test case is in the same file and run one after the other.

Reply:
what should that comment be ?